packages/sample-apps/windows/SampleAppCS/SampleAppCS.csproj

```diff
@@ -16,7 +16,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
```

Comment:
App projects shouldn't hardcode these versions, but instead follow this pattern from the template (after the setting of ReactNativeWindowsDir and the top PropertyGroup):

```xml
<Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.WindowsSdk.Default.props" />
```